crates/iota/tests/cli_tests.rs

```diff
@@ -3217,7 +3214,7 @@ async fn test_stake_with_u64_amount() -> Result<(), anyhow::Error> {
         "0x5",
         &format!("[{}]", coins.first().unwrap().coin_object_id),
         "[1000000000]",
-        &validator_addr.to_string(),
+        &committee_member_addr.to_string(),
```

Comment:
I think it would actually be useful to also keep one test that stakes to a validator that is not a committee member, so this is still covered

Reply:
That's a good point, but for this issue I think we need to go through all the other places where we wanna test committee member and active validators and add the associated testings at one time, maybe in another PR.